android/src/main/java/tv/vizbee/rnsender/VizbeeCastButtonViewManager.java

```diff
@@ -21,4 +24,10 @@ protected VizbeeCastButtonView createViewInstance(ThemedReactContext reactContex
     public void onDropViewInstance(VizbeeCastButtonView buttonView) {
         super.onDropViewInstance(buttonView);
     }
+
+    @ReactProp(name = "color")
+    public void setColor(VizbeeCastButtonView view, @Nullable String color) {
```

Comment:
Why the color property annotation is @Nullable? Shouldn't it be @NonNull considering the method itself is to color the cast icon?